packages/backend/migrations/20241119125038_add_unique_constraint_to_kid.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  return knex
+    .raw(
+      // Keep only one active walletAddressKey (the most recent row)
+      `DELETE FROM "walletAddressKeys" w
+       WHERE revoked = false
+       AND ctid NOT IN (
+          SELECT MIN(ctid)
+          FROM "walletAddressKeys"
+          WHERE revoked = false
+            AND kid = w.kid
```

Comment:
I think we need to group by `walletAddressId` and `kid` here still, otherwise we may delete keys that have the same `kid` but belong to a different wallet address.

Additionally, we should also use `MAX(ctid)` so we can grab the most recent key